src/Components/TwinPropertySearchDropdown/TwinPropertySearchDropdown.tsx

```diff
@@ -143,6 +144,7 @@ const TwinPropertySearchDropdown = (
 
             twinSearchContinuationToken.current =
                 searchTwinAdapterData.adapterResult.result.data.continuationToken;
+            options.sort(sortAscendingOrDescending('label'));
```

Comment:
nit: can you move this up above line 143 so the value that gets stored to state is sorted?